boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -17,9 +17,9 @@ The nRF5340 is a dual-core SoC based on the Arm® Cortex®-M33 architecture, wit
 * a secondary ARM Cortex-M33 core, with a reduced feature set, running at
   a fixed 64 MHz, referred to as the **Network MCU**.
```

Comment:
```suggestion
  a fixed 64 MHz, referred to as the **network core**.
```
Same change for Application MCU -> application core, and both throughout the document.

Reply:
updated now, @ru-fu